.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--- PR title format: [GH#<Issue ID>][BZ<BZ ID>][<Jira ID>]: <Short description of your PR> --->
+
+
+Version:
+<!--- What version of OpenShift does this apply to? --->
+
+
+Issue:
+<!--- Add a link to your Bugzilla, Jira, or GitHub issue. --->
+
+
+Docs Preview:
+<!--- Add the direct link to your content. --->
+
+Summary of changes
+*
+
+
+Review checklist
```

Comment:
This is a great idea! Would a checkmark indicate that a review is required or that it was completed?

Reply:
My initial thought was that the author could add a check mark as the PR went through the review process. 

But it is ambiguous.